src/components/settings/GcodePreviewSettings.vue

```diff
@@ -131,6 +131,18 @@
 
       <v-divider />
 
+      <app-setting :title="$t('app.setting.label.auto_load_mobile_on_print_start')">
+        <v-switch
+          v-model="autoLoadMobileOnPrintStart"
+          hide-details
+          class="mb-5"
+          @click.native.stop
+          :disabled="!autoLoadOnPrintStart"
```

Comment:
For viz. I've added a disabled state to the slider that's dependent on `autoLoadOnPrintStart` being enabled

Reply:
In these cases, we prefer to just hide the control!

Reply:
Is that done with a simple v-if or is there some animation I have to consider?

Reply:
`v-if` it please!